src/VerifyU2FController.js

```diff
@@ -42,7 +41,7 @@ function (Okta, FormController, FormType, FooterSignout, Q, HtmlErrorMessageView
   }
 
   return FormController.extend({
-    className: 'verify-u2f',
+    className: 'mfa-verify verify-u2f',
```

Comment:
Curious, why we need to add this class name `mfa-verify`

Reply:
It was missing. In theory our verify views should have it (https://github.com/okta/okta-signin-widget/blob/master/assets/sass/okta-theme.scss#L796)